byron/semantics/executable-spec/src/Control/State/Transition/Trace.hs

```diff
@@ -428,3 +409,34 @@ extractValues d =  catMaybes (gmapQ extractValue d)
   where
     extractValue :: forall d1 . (Data d1) => d1 -> Maybe a
     extractValue d1 = cast d1
+
+data STTriple a =
```

Comment:
Why not call this `SourceSignalTarget`? I think it represents the type better, and besides you're [German](https://chrisdone.com/posts/german-naming-convention/) :P

Reply:
contrary to popular belief - as usual - most German words are as short as possible and often abbr'vd :-)